src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './CurrentUser.scss';
+import { getUser } from '../../api/api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number,
+  clear: () => void,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({ userId, clear }) => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [hasLoadingError, setHasLoadingError] = useState(true);
+
+  useEffect(() => {
+    try {
+      getUser(userId)
+        .then((user: User) => setSelectedUser(user));
+    } catch (error) {
+      setHasLoadingError(false);
+    }
```

Comment:
we use try {} catch {} with async await
and catch() after then() with then() functionality, rewrite it